build.gradle

```diff
@@ -432,7 +432,7 @@ ext {
             ],
             graal                      : [
                     version: graalVersion,
-                    group  : 'com.oracle.substratevm',
+                    group  : 'org.graalvm.nativeimage',
```

Comment:
I don't think we can remove the old one without breaking existing apps

Reply:
They have only changed the artifact coordinates but not the package for any class. Keep also in mind that this dependency is used as `compileOnly` when adding the `graal-native-image` feature in the CLI.
How do you suggest to do this change then?